llvm/test/Instrumentation/MemorySanitizer/AArch64/qshrn.ll

```diff
@@ -0,0 +1,750 @@
+; NOTE: Assertions have been autogenerated by utils/update_test_checks.py UTC_ARGS: --version 4
+; RUN: opt < %s -passes=msan -S | FileCheck %s
+;
+; Forked from llvm/test/CodeGen/AArch64/qshrn.ll
```

Comment:
Add a comment that they are currently incorrectly handled?

Reply:
Done